### PR TITLE
Ci fixes

### LINKED ISSUE #13: concurrent image builds fail due to instance name conflicts
If two images are built at the same time, they currently will often fail due to the CI image names conflicting. That's because I (andres) made the image names predictable, to avoid a lot of instances building up in case of failure. But it's likely more confusing than it's worth.  Perhaps we can give instances a shorter max lifetime? And then name them with a recognizable prefix, but a random postfix?

### LINKED ISSUE #14: reduce redundant references to freebsd version number
As seen in 3c43073ff4a48790aba59039f3c4a9fbb969bc63 there's multiple references to freebsd-13-$n in the packer file. It'd be good to reduce that to exactly one.

### PR DESCRIPTION
**Pull request by nbyavuz**
Fixes #13, #14.

### CODE REVIEW

File: packer/freebsd.pkr.hcl [hunk 1]
```diff
@@ -48,7 +49,7 @@ build {
 
       zone = tag.value.zone
       machine_type = tag.value.machine
-      instance_name = "build-${local.name}-${tag.value.name}"
+      instance_name = "build-${local.name}-${tag.value.name}-${var.image_date}"
```
Comment (anarazel) [inline packer/freebsd.pkr.hcl:52]: What format does image_date have? It seems this either could lead to duplicates if it really is just the date, or to too long instance names (IIRC they're limited to 63 chars or such)

Comment (anarazel) [reply packer/freebsd.pkr.hcl:52]: > It is the same DATE variable which is used in image names and it is like 2022-08-01t14-20

Ok. I guess it'd be better to add seconds for this purpose?

> For now, the longest name of the instance is build-pg-ci-sid-newkernel-uring-2022-08-01t14-20 which is 48 chars.

Perhaps we could remove "pg-ci-" from the image name?

Comment (nbyavuz) [reply packer/freebsd.pkr.hcl:52]: > > It is the same DATE variable which is used in image names and it is like 2022-08-01t14-20
> 
> Ok. I guess it'd be better to add seconds for this purpose?

Yes, that could work.

> 
> > For now, the longest name of the instance is build-pg-ci-sid-newkernel-uring-2022-08-01t14-20 which is 48 chars.
> 
> Perhaps we could remove "pg-ci-" from the image name?

Are we removing "pg-ci" from image names or instance names?

Comment (anarazel) [reply packer/freebsd.pkr.hcl:52]: > Are we removing "pg-ci" from image names or instance names?

err, instance, not image names.

Comment (nbyavuz) [reply packer/freebsd.pkr.hcl]: Done.

**Review by anarazel — APPROVED** [review]

**Pull request merged**